kubernetes-maven-plugin/doc/src/main/asciidoc/inc/_introduction.adoc

```diff
@@ -237,31 +238,10 @@ A working example can be found in the  https://github.com/jkubeio/kubernetes-mav
       </all>
     </labels>
 
-    <deployment> <!--4-->
-      <name>${project.artifactId}</name>
-      <replicas>1</replicas>
-
-      <containers> <!--5-->
-        <container>
-          <alias>camel-app</alias> <!--6-->
-          <ports>
-            <port>8778</port>
-          </ports>
-          <mounts>
-            <scratch>/var/scratch</scratch>
-          </mounts>
-        </container>
-      </containers>
-
-      <volumes> <!--7-->
-        <volume>
-          <name>scratch</name>
-          <type>emptyDir</type>
-        </volume>
-      </volumes>
-    </deployment>
-
-    <services> <!--8-->
+    <replicas>2</replicas> <!--4-->
+    <controllerName>${project.artifactId}</controllerName> <!--5-->
+
+    <services> <!--6-->
```

Comment:
What happened to these configurations, why have we lost support for this?

Reply:
I looked for old plugin versions but I couldn't find any config for containers or deployments.